### DISCUSSION TITLE
unable to create a new hobby team

### DISCUSSION THREAD

**Discussion by laylatichy** [vercel/community#6725] [Account Issue]
### Question

so I had 2 hobby teams, I had to delete 1 of them few days ago as it was no longer needed and will need to create a new one soon for other projects but Im unable to do so

https://vercel.com/docs/accounts/plans/pro/trials#starting-a-trial

from the docs

![image](https://github.com/vercel/community/assets/52243892/dce2596e-65b7-4d5f-bb9f-bd5729cefcd5)

but my team creation screen is forcing me to pay for a pro plan

![image](https://github.com/vercel/community/assets/52243892/b15e6ac8-e0b9-45e6-8bb9-531bf6283e89)

is that how it should be or something wrong here?

I can create pro team and downgrade it to hobby, just wanted to avoid being charged for it for no reason

**Comment by leerob** [comment]
Question: Is there a reason not to use your existing hobby team?

**Reply by laylatichy** [reply]
mostly because as as freelancer setting up teams for various clients then transferring an ownership is far easier and streamlined, it would be a bigger hassle to get the client to create a vercel account then tell them to invite me to their team and transfer a project o rmultiple projects over instead of just inviting them to the team and changing owner for a handout

plus its far easier invoicing wise, I can just forward them an invoice from vercel for a specific team

I mean if thats how it supposed to work I'm fine with it I can create pro team then just downgrade it, was just surprised I was unable to create a hobby tier team no more, and I had a few of them running while back, but they were created around the same time and most of them are transferred over to the clients now

its just that there is no information anywhere that you can have only 1 hobby team, unless Im missing something?

even on the limits page I see

![image](https://github.com/vercel/community/assets/52243892/1d939475-9d61-4fc0-9a91-3752c1c4a276)

so I hope you understand that its a bit confusing

**Comment by awmartin** [comment]
I can't even create a second project on my single hobby team. I'm thoroughly confused by Vercel so far.

**Reply by laylatichy** [reply]
welcome to vercel I guess, I worked around it by creating new account, creating a new team with projects there, then transferring it back to my primary account

**Reply by amyegan** [reply]
~~Hey @awmartin. There are a few reasons someone might run into trouble creating a new project. Can you walk me through what you tried and what you see on the screen? That will help me understand exactly what is going wrong in your case.~~

_Update:_ responded in https://github.com/vercel/community/discussions/7246

[a comment by an automated account is not shown]
